### DISCUSSION TITLE
# 玩一把爽一年的存档

### DISCUSSION THREAD

**Discussion by Philia093-T** [share121/inter-knot#1666] [其他]
本来以为只是一次普通的模拟
结果第二间给我来了一个天才八卦，然后又是分裂银币又是效果为双倍碎片的塔罗牌

---
<img width="1280" alt="Honkai  Star Rail Screenshot 2024 08 12 - 13 28 49 49" src="https://github.com/user-attachments/assets/f1c578f8-cb2f-4596-a40a-ebad1c2785a0">
---

第一层boss间给了个鲁伯特，下一间冒险关又给两个鲁伯特，差分机直接出来了
感觉哪怕开挂都不会这样来
最后附上最终间的情况
<img width="1280" alt="Honkai  Star Rail Screenshot 2024 08 12 - 13 47 23 30" src="https://github.com/user-attachments/assets/4a37d0db-bb7d-411e-b512-a7d93d4ed066">

---

<img width="1280" alt="Honkai  Star Rail Screenshot 2024 08 12 - 13 47 28 38" src="https://github.com/user-attachments/assets/1ca7182e-16cf-40b0-922c-ebd1a8d9e8af">

---
**这把存档能不能不删啊，我什么都会做的**😭

**Comment by Philia093-T** [comment]
原来是会拿第一张图片做封面的吗

**Comment by DUSK42** [comment]
五毛我玩

**Comment by BSNHD** [comment]
这三个奇物和一块这么逆天，我都还没拿到过

**Reply by Philia093-T** [reply]
真的，拿到就是爽局，我感觉阮梅都不一定比得过这个
